### PR TITLE
FIX: Include part size sum in Utapi object

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Get sum of part sizes from existing part objects to pass to utapiClient.

### CODE REVIEW

File: lib/api/multipartDelete.js [hunk 1]
```diff
@@ -117,6 +120,7 @@ function multipartDelete(authInfo, request, log, callback) {
                     pushMetric('abortMultipartUpload', log, {
                         authInfo,
                         bucket: bucketName,
+                        byteLength: partSizeSum,
```
Comment (LaurenSpiegel) [inline lib/api/multipartDelete.js]: Is this turned into a negative number somewhere?

Comment (bennettbuchanan) [inline lib/api/multipartDelete.js]: No. UtapiClient will use the value to decrement the `storageUtilized` metric. Similar to how DELETE operations are handled. 😄

Comment (LaurenSpiegel) [inline lib/api/multipartDelete.js]: Great.  Can you get this ready to merge?

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
e2e passed

**Pull request merged**